node/src/components/consensus/era_supervisor.rs

```diff
@@ -204,8 +220,12 @@ where
     fn booking_block_height(&self, era_id: EraId) -> u64 {
         // The booking block for era N is the last block of era N - AUCTION_DELAY - 1
         // To find it, we get the start height of era N - AUCTION_DELAY and subtract 1
-        let after_booking_era_id =
-            EraId(era_id.0.saturating_sub(self.protocol_config.auction_delay));
+        let after_booking_era_id = EraId(
+            era_id
+                .0
+                .saturating_sub(self.protocol_config.auction_delay)
+                .max(self.protocol_config.last_activation_point.0),
+        );
```

Comment:
We should implement all those traits and functions for `EraId` (a lot can be derived with `derive_more`):
```suggestion
        let after_booking_era_id =  era_id
                .saturating_sub(self.protocol_config.auction_delay)
                .max(self.protocol_config.last_activation_point);
```

Reply:
Maybe I'll create a ticket for that, because I think it includes some design work as well (like a decision whether we implement `EraId + EraId`, or only `EraId + u64`, for example) - would that be okay?

Reply:
Sure… but definitely _not_ `EraId + EraId`, only the latter.